### DISCUSSION TITLE
UniFi Network Application LXC

### DISCUSSION THREAD

**Discussion by killerrene** [tteck/Proxmox#504] [Help Needed] [answered]
Can you help with: for console, I can't find login and username and password. As it is set up with

**Comment by tteck** [comment, marked as answer]
If it's asking for a user and password in the console, and you didn't assign a password during install, the script didn't finish. Remove and start over.

**Comment by DobleJSoto** [comment]
Hello, I take advantage of this thread to warn you that a clean installation of the unifi container offers 2 backups to restore, I have not restored them so I do not know what they contain.
